pip-requirements.txt

```diff
@@ -0,0 +1,2 @@
+mock>=2.0.0
+coveralls
```

Comment:
@OwenArnold Agreed. These all seem like they don't need to be solved in this PR though? It is getting rather large.